loops.js

```diff
@@ -32,9 +39,15 @@
 
 // Tip: een getal is oneven als je het door 2 deelt en er blijft nog iets over. Een getal is even als je het deelt door 2 en er blijft niets over..
 // ==========================================
-
-
-
+const even = " is even"
```

Comment:
Ook hier heb je de extra constanten niet per se nodig 

for (let i = 0; i < 16; i++) {
  if (i % 2 === 0) {
    console.log(i + ' is even');
  }
  if (i % 2 !== 0) {
    console.log(i + ' is oneven');
  }
}